.github/workflows/update_docsstrings_tutorials.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install pydoc-markdown==3.11.0
+          pip install pydoc-markdown
```

Comment:
I guess we would still like to pin the version but now to 4.x? Without a pinned version things might break in future with the next release and we would need to search for the cause of the problem.

Reply:
Good point :+1: